skeleton/scaffold-minimum/test__if_karma_or_jest/unit/app.spec.ext

```diff
@@ -1,7 +1,32 @@
+import {bootstrap} from 'aurelia-bootstrapper';
 import {App} from '../../src/app';
+import {StageComponent} from 'aurelia-testing';
 
 describe('the app', () => {
```

Comment:
We probably should remove this duplicated test now, plus the static import above. Showing StageComponent in demo test code is better and more relevant.

Reply:
Removed test